cypress/tests/dynamic-scan.spec.ts

```diff
@@ -259,23 +262,27 @@ describe('Dynamic Scan', () => {
 
                 if (type !== 0) {
                   // change to any device
-                  cy.get('@deviceTypeSelect').click({ force: true });
-
-                  cy.document()
-                    .findByRole('option', {
-                      name: cyTranslate('anyDevice'),
-                    })
-                    .should('exist')
-                    .click({ force: true });
-
-                  cy.findByRole('button', {
-                    name: cyTranslate('anyDevice'),
-                  }).should('exist');
-
-                  cy.document()
-                    .findByText(cyTranslate('savedPreferences'))
-                    .should('exist');
+                  dynamicScanActions.chooseDevicePreferenceOption(
+                    '@deviceTypeSelect',
+                    cyTranslate('anyDevice')
+                  );
                 }
+
+                if (version !== '0') {
```

Comment:
Here also. What does the `'0'` check stand for?

Reply:
changed it to a constant `ANY_VERSION`